spec/statesman/adapters/active_record_queries_spec.rb

```diff
@@ -15,7 +15,14 @@
 
   before do
     MyActiveRecordModel.send(:include, Statesman::Adapters::ActiveRecordQueries)
+    MyActiveRecordModel.send(:has_many,
```

Comment:
It makes sense to me:) but I don't know/understand what I should do in particular. I'm sorry.